lib/routers/profile/permission.js

```diff
@@ -48,7 +48,7 @@ const removable = () => (req, res, next) => {
   };
   Profile.scopeSingle(params).get()
     .then(profile => {
-      const hasProjects = profile.projects.filter(project => project.status === 'active').length;
+      const hasProjects = profile.projects.filter(project => project.establishmentId === req.establishment.id && project.status === 'active').length;
```

Comment:
```suggestion
      const hasProjects = profile.projects
        .filter(project => project.establishmentId === req.establishment.id || project.additionalEstablishments.find(e => e.id === req.establishment.id))
        .filter(project.status === 'active')
        .length;
```

Reply:
Note: this is 100% untested and made up by me in the github edit window, so you might want to run it first.